datastore/src/test/java/io/spine/server/storage/datastore/given/TestEnvironment.java

```diff
@@ -59,10 +59,18 @@ public static boolean runsOnCi() {
         return onCi;
     }
 
+    /**
+     * Returns a new specification of a multi-tenant {@code BoundedContext} named after
```

Comment:
I think, "multitenant" is a single word.

Reply:
I think, not necessarily.

Please see [this](https://en.wikipedia.org/wiki/Talk%3AMultitenancy), [this](https://www.cisco.com/c/en/us/td/docs/switches/datacenter/dfa/configuration/b-dfa-configuration/multi-tenancy.pdf), and many others.

The idea I have here is that "multitenancy" is a long word, and in scope of our documentation it's much easier for eyes to catch this hyphen in "multi-tenancy", rather than parse this "..ltit.." in the middle of the long version.